messaging/src/main/java/org/jboss/as/messaging/TransportConfigOperationHandlers.java

```diff
@@ -145,6 +145,16 @@ static void processConnectors(final OperationContext context, final Configuratio
                 final ModelNode config = property.getValue();
                 final Map<String, Object> parameters = getParameters(context, config);
                 final String clazz = FACTORY_CLASS.resolveModelAttribute(context, config).asString();
+                if (NettyConnectorFactory.class.getName().equals(clazz)) {
```

Comment:
yes I understand. the problem happens when socket-binding references a outbound socket binding, like
<outbound-socket-binding name="messaging-remote-1" fixed-source-port="true">
            <remote-destination host="127.0.0.1" port="5545"/>
        </outbound-socket-binding>
it won't get passed in.